CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 28, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Aggregation (#5592) @flaviendelangle
+
+  Premium users can now aggregate data in the grid.
+  You can now use aggregation functions, and with a couple of clicks, extract information like sum, average, count, and others
+
+  ![modify aggregation](https://user-images.githubusercontent.com/45398769/181478185-142f87b0-c8de-4585-b381-cf4fdc62d99f.gif)
+
+  To enable the feature, add `experimentalFeatures={{ aggregation: true }}`.
+  Aggregation function can be customized, and interact with row aggregation.
+  See the [documentation](https://mui.com/x/react-data-grid/aggregation/) to explore all the customization options.
+
+- 🚀 Row pinning (#4863) @cherniavskii
```

Comment:
?
```suggestion
- 🚀 Introduce row pinning (#4863) @cherniavskii
```

Reply:
We also need a detailed highlight for this one.

Reply:
I would rather just name the features in the highlights, as we did in https://github.com/mui/mui-x/releases/tag/v5.11.0

> We also need a detailed highlight for this one.

There's description already, but I can add a screenshot or a video from https://deploy-preview-33595--material-ui.netlify.app/blog/aggregation-functions/#row-pinning
What do you think?

Reply:
We used to name highlights for big features differently. I guess this changed some time ago. `Introduce row pinning` sounds more human in comparison to just the feature name.

Regarding the highlight - I think it does deserve a gif. It's an important feature and it is also very visual.

Reply:
I'll add it as soon as we decide whether to use GIF or video